nixos/modules/services/networking/routinator.nix

```
@@ -0,0 +1,71 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.routinator;
+  settingsFormat = pkgs.formats.toml { };
+in
+{
+  options.services.routinator = {
+    enable = mkEnableOption "Routinator 3000 is free, open-source RPKI Relying Party software made by NLnet Labs.";
+
+    settings = mkOption {
+      inherit (settingsFormat) type;
+      example = literalExpression ''
+
+        {
+          repository-dir = "/var/lib/routinator/rpki-cache";
+          rtr-listen = [ "127.0.0.1:3323" ];
+          http-listen = [ "127.0.0.1:8323" ];
+        }
+      '';
+      description = ''
+
+        Configuration for Routinator 3000, see <https://routinator.docs.nlnetlabs.nl/en/stable/manual-page.html#configuration-file> for options
+      '';
+      default = {
+        repository-dir = "/var/lib/routinator/rpki-cache";
+        rtr-listen = [ "127.0.0.1:3323" ];
```

I would expect the primary use case here to expose RPKI-RTR to other BGP speakers on the network, so defaulting to dual-stack wildcard is probably fine.

```suggestion
        rtr-listen = [ "[::]:3323" ];
```

---

Probably yes. I would personally leave it that way as according to the Routinator Docs it's the default config shipped in other distros packages.
https://routinator.docs.nlnetlabs.nl/en/stable/configuration.html#setup-when-installed-from-a-package

---

Fine with me.